test/nats-e2e/testdata/nats-auth-fake-token.yaml

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Is this file in-use?

Reply:
yes it is used here : //go:generate kubectl apply -f testdata/nats-auth-fake-token.yaml -n numaflow-system